src/backend/marsha/core/api.py

```diff
@@ -39,28 +39,26 @@ def update_state(request):
         The request on the API endpoint, it should contain a payload with the following fields:
             - key: the key of an object in the source bucket as delivered in the upload policy,
             - state: state of the upload, should be either "ready" or "error",
-            - signature: has of the payload salted with a shared secret to authenticate AWS.
+            - extraParameters: Dict containing arbitrary data sent from AWS Lambda.
 
     Returns
     -------
     Type[rest_framework.response.Response]
         HttpResponse acknowledging the success or failure of the state update operation.
 
     """
+    msg = request.body.decode("utf-8")
```

Comment:
Decoding seems useless because it is re-encoding it where it is being used below...